integrations/observability/grafana/install.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+set -o errexit
+set -o xtrace
+
+kubectl apply -f ${SCRIPT_DIR}/resources.yaml
+
+apiserver_patch_file=$(mktemp)
+cat <<EOF > ${apiserver_patch_file}
+---
+apiVersion: v1
+kind: Deployment
+spec:
+  template:
+    spec:
+      containers:
+        - name: kubearchive-api-server
+          env:
+            - name: KUBEARCHIVE_OTEL_MODE
+              value: "enabled"
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: "http://otel-collector.observability.svc.cluster.local:4318"
+EOF
+kubectl -n kubearchive patch deployment kubearchive-api-server --patch-file ${apiserver_patch_file}
```

Comment:
Consider using the [kubectl set env command](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_set/kubectl_set_env/) for doing this:
```suggestion
kubectl -n kubearchive rollout pause deploy/kubearchive-api-server
kubectl -n kubearchive set env deploy/kubearchive-api-server KUBEARCHIVE_OTEL_MODE=enabled
kubectl -n kubearchive set env deploy/kubearchive-api-server OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector.observability.svc.cluster.local:4318
kubectl -n kubearchive rollout resume deploy/kubearchive-api-server
```

Reply:
Nice! I wasn't aware that `set env` could be used for more than one variable at a time